osu.Game/Screens/Multiplayer/Components/RoomInspector.cs

```diff
@@ -299,6 +323,36 @@ protected override void UpdateAfterChildren()
             participantsScroll.Height = DrawHeight - topFlow.DrawHeight;
         }
 
+        private void nullState()
```

Comment:
i don't like these methods. instead, make an `updateState` method which performs *all* state changes and call it from the setter and anywhere else a state update is required. see `osu.Game.Overlays.BeatmapSet.Info` and all other overlay relevant classes for examples to follow.